packages/strapi-parts/src/Loader/Loader.stories.mdx

```diff
@@ -10,14 +11,27 @@ import { Loader } from './Loader';
 
 # Loader
 
-This is the doc of the `Loader` component
+Loaders indicate some data is currently loading where content usually takes place. They help to notify users that loading is ongoing.
 
-## Loader
+**Best** **practices**
 
-Description...
+- Once the content appears, the loader disappears.
+- If the trigger only concerns a portion of a page, the loader should be placed in that part.
+- Loaders should be scaled down if they aimed to be placed in a small portion of a page.
+- Use a loader whenever the wait time is to be longer than 1 second.
+
+
+## Usage
+
+Even if loaders give an indication that an action has been processed, they do not provide details about what is happening on the back-end side. 
```

Comment:
I personally think this is not a valid rule. As a user, I want to know that my locales are resolving (and not my permissions somehow). Loading should be contextualized, to me

Reply:
But it's not the case, right?

Reply:
It's the case, but it's hidden. In this scenario, people using assistive technologies have more feedbacks than people using a screen monitor 😊 . But as mentioned, that's a personal take. But still, a label should be provided, even if hidden, for people using ATs

Reply:
@mfrachet Let's talk about this directly, so we can make a decision quickly :)

Reply:
@maevalienard Has a decision been made?

Reply:
Not yet 😂 Let me bother you instead :p